verbose: False
debug: False

detector:
  model_id: opencv_dnn

  opencv_cascade:
    model: haarcascade_frontalcatface.xml
    scale_factor: 1.3
    min_neighbors: 5
    height: 300
    cuda: False

  opencv_dnn:
    model: caffe #tensorflow or caffe
    threshold: 0.5
    scale_factor: 1.0
    height: 300
    mean: [104, 117, 123]

  dlib_hog:
    model: default
    height: 300 #width is proportional

  dlib_mmod:
    model: mmod_human_face_detector.dat
    height: 300 #width is proportional

aligner:
  model_id: openface #openface

  openface: 
    model: shape_predictor_68_face_landmarks.dat
    image_dimension: 96

embosser:
  model_id: openface #openface

  openface:
    model: nn4.small2.v1.t7
    image_dimension: 96
    cuda: False
  
  vgg_face:
    model: VGG_FACE.t7
    cuda: true

classifier:
  model_id: sklearn #sklearn

  sklearn:
    model: classifier_MLP_emilly.pkl
    threshold: 0.80
